docs/community/contributing.md

```diff
@@ -0,0 +1,275 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Contribution Guidelines
+
+**Apache Hadoop Submarine** is an [Apache2 License](https://github.com/apache/hadoop-submarine/blob/master/LICENSE) Software.
+
+Contributing to Hadoop Submarine (Source code, Documents, Image, Website) means you agree to the Apache2 License.
```

Comment:
Usually a contributor should submit a Contributor Agreement as well: https://www.apache.org/licenses/contributor-agreements.html

Reply:
I have consulted this question, Apache 2.0 license, you can not sign Contributor Agreement.